### PR TITLE
Milestone 3: Blog App | Validation model specs

### PR DESCRIPTION
**Pull request by michael-duke**
## Blog App milestone 3 : Validation Model Specs

In this milestone I have:

- Used `validations` for models.
- Written `unit tests` for models.
- Implemented unit specs for all of your `models'` methods and `validations`.

can-do-this-all-day

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -10,6 +16,11 @@ def recent_five
 
   private
 
+  def set_defaults
+    self.comments_counter = 0
+    self.likes_counter = 0
+  end
```
Comment (fobadara) [inline app/models/post.rb]: [OPTIONAL] You don't need this here. You could have just set it in the migrations

**Review by fobadara — APPROVED** [review]
Hi @michael-duke, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters are passing
✅ Good first attempt

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/post.rb (hunk 1 above)
Comment (michael-duke) [reply app/models/post.rb]: Yes @fobadara, I will make that change in the migrations. I simply wanted to try something new 😄

**Comment by michael-duke** [comment]
Thank you @fobadara for you feedback, happy coding 😄 🚀

**Pull request merged**